Remove setting -D_FORTIFY_SOURCE=2, as Gentoo's GCC already
enables it by default, causing unnecessary churn:

* <command-line>:0:0: warning: "_FORTIFY_SOURCE" redefined

--- a/configure.ac
+++ b/configure.ac
@@ -400,7 +393,6 @@
 	CXXFLAGS="$CXXFLAGS -Wall -Wextra -Wcast-align -Wshadow -Wwrite-strings -Wctor-dtor-privacy -Wnon-virtual-dtor -Wreorder -Wsign-promo -Wundef " # -Wcast-qual -Wbad-function-cast -Wwrite-strings -Woverloaded-virtual -Wmissing-declarations
 
 	XIPH_ADD_CFLAGS([-Wdeclaration-after-statement])
-	XIPH_ADD_CFLAGS([-D_FORTIFY_SOURCE=2])
 
 	AC_LANG_PUSH([C++])
 	XIPH_ADD_CXXFLAGS([-Weffc++])
